integration-tests/src/test/java/integration/MongoDbSeedRule.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package integration;
+
+import integration.util.graylog.ServerHelper;
+import integration.util.mongodb.MongodbSeed;
+import org.junit.rules.MethodRule;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+
+import static org.apache.commons.lang3.ObjectUtils.firstNonNull;
+
+public class MongoDbSeedRule implements MethodRule {
```

Comment:
We cannot use nosql-unit to seed two engines in one test, but we have tests that require both seed data in MongoDB _and_ ES.